packages/react/src/lib/accessibility/Behaviors/Alert/alertProps.ts

```diff
@@ -0,0 +1,10 @@
+type AlertProps = {
+  /** An alert may be formatted to display a danger message. */
+  danger?: boolean
+  /** An alert may be formatted to display a warning message. */
+  warning?: boolean
+  /** id of the alert body element. */
+  bodyId?: string
+}
+
+export default AlertProps
```

Comment:
Can you please clarify why we need to move out types?..

Reply:
Because both `alertBehavior.ts` and `alertWarningBehavior.ts` depends on it.

Reply:
Can we keep in `alertBehavior.ts`?

Reply:
I added to to `alertWarningBehavior` to avoid circular dependency (`alertBehavior` imports `alertWarningBehavior`)